Formula/mit-scheme.rb

```diff
@@ -0,0 +1,119 @@
+  url "http://ftpmirror.gnu.org/mit-scheme/stable.pkg/9.2/mit-scheme-c-9.2.tar.gz"
+  mirror "https://ftp.gnu.org/gnu/mit-scheme/stable.pkg/9.2/mit-scheme-c-9.2.tar.gz"
+  sha256 "4f6a16f9c7d4b4b7bb3aa53ef523cad39b54ae1eaa3ab3205930b6a87759b170"
+  revision 1
+
+  bottle do
+    revision 1
+    sha256 "1a449d9c1f1043c1a557962332c7ee18664f5f15ea9eca8e754c0c3cb3c4aef4" => :el_capitan
+    sha256 "7966d841f92346a68ee088b0e25394e541b690aafbb306e27b352ee35a1e24c7" => :yosemite
+    sha256 "a782017f01f5c385e343c802a33cd634b1cfd89ec7eed94a43d753034eaa15ab" => :mavericks
+  end
+
+  conflicts_with "tinyscheme", :because => "both install a `scheme` binary"
+
+  # Has a hardcoded compile check for /Applications/Xcode.app
+  # Dies on "configure: error: SIZEOF_CHAR is not 1" without Xcode.
+  # https://github.com/Homebrew/homebrew-x11/issues/103#issuecomment-125014423
+  depends_on :xcode => :build
+  depends_on "openssl"
+  depends_on :x11 => :optional
+
+  def install
+    # Setting -march=native, which is what --build-from-source does, can fail
+    # with the error "the object ..., passed as the second argument to apply, is
+    # not the correct type." Only Haswell and above appear to be impacted.
+    # Reported 23rd Apr 2016: https://savannah.gnu.org/bugs/index.php?47767
+    # Note that `unless build.bottle?` avoids overriding --bottle-arch=[...].
+    ENV["HOMEBREW_OPTFLAGS"] = "-march=#{Hardware.oldest_cpu}" unless build.bottle?
+
+    # The build breaks __HORRIBLY__ with parallel make -- one target will
+    # erase something before another target gets it, so it's easier to change
+    # the environment than to change_make_var, because there are Makefiles
+    # littered everywhere
+    ENV.deparallelize
+
+    # Liarc builds must launch within the src dir, not using the top-level
+    # Makefile
+    cd "src"
+
+    # Take care of some hard-coded paths
+    %w[
+      6001/edextra.scm
+      6001/floppy.scm
+      compiler/etc/disload.scm
+      edwin/techinfo.scm
+      edwin/unix.scm
+      swat/c/tk3.2-custom/Makefile
+      swat/c/tk3.2-custom/tcl/Makefile
+      swat/scheme/other/btest.scm
+    ].each do |f|
+      inreplace f, "/usr/local", prefix
+    end
+
+    inreplace "microcode/configure" do |s|
+      s.gsub! "/usr/local", prefix
+      # Fixes "configure: error: No MacOSX SDK for version: 10.10"
+      # Reported 23rd Apr 2016: https://savannah.gnu.org/bugs/index.php?47769
+      s.gsub! /SDK=MacOSX\${MACOSX}$/, "SDK=MacOSX#{MacOS.sdk.version}"
+    end
+
+    if build.without? "x11"
+      inreplace "etc/make-liarc.sh" do |s|
+        s.gsub! "run_configure", "run_configure --without-x"
+      end
+    end
+
+    # The configure script will add "-isysroot" to CPPFLAGS, so it didn't
+    # check .h here by default even Homebrew is installed in /usr/local. This
+    # breaks things when gdbm or other optional dependencies was installed
+    # using Homebrew
+    ENV.prepend "CPPFLAGS", "-I#{HOMEBREW_PREFIX}/include"
```

Comment:
I'm having some trouble figuring out what those individual dependencies are. The comment above mentions `gdbm`, but it's not specified in the formula itself. The only dependencies actually mentioned in the formula don't have `include`s in `/usr/local`, so I'm not sure this line is actually performing anything.